helpdesk/urls.py

```diff
@@ -215,7 +215,7 @@ def get_context_data(self, **kwargs):
     path(
         "logout/",
         auth_views.LogoutView.as_view(
-            template_name="helpdesk/registration/login.html", next_page="../"
+            template_name="helpdesk/registration/login.html", next_page=reverse_lazy("helpdesk:home")
```

Comment:
Note that the `template_name` does not seem to be useful when we provide `next_page`.  
Moreover, there is a template for the [confirmation of logging out](https://github.com/django-helpdesk/django-helpdesk/blob/main/helpdesk/templates/helpdesk/registration/logged_out.html), but it's not called anywhere. I believe it would make more sense to point to it instead of login page.

Reply:
Agreed - go ahead and change the logout next page to the confirmation of logout page.

Reply:
Ideally the logged out page needs a login button otherwise the user who is security concious and logs out regularly has to change the URL or use a bookmark.

Reply:
Alright, here's what I have now for the logged_out page :

![image](https://github.com/django-helpdesk/django-helpdesk/assets/15046586/844cbb9a-f329-4e10-80a2-071b89d56989)

What do you think?

A login button is available in the header so it might not be needed to show it again.